code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -92,6 +92,13 @@
 /turf/open/return_analyzable_air()
 	return return_air()
 
+/turf/open/proc/has_gas(datum/gas, mole_amount)
```

Comment:
1: Why is this on /turf/open instead of /datum/gas_mixture?
2: The gas argument should not be typed as a datum, it's a typepath.

Reply:
So my thought process went like this:

I need a way to check if a turf has a certain moles of gas.  TURF -> GASES -> GAS
Thus the root class that I thought I needed was Turf.  It appears /datum/gas_mixture is a global class used for all things.  It might be useful to have the proc inside that class instead.  Let me tinker with it since I'm still pretty new to coding in DM.

Reply:
Both of these are resolved with 73324af